it/itUtils/ViewTest.scala

```diff
@@ -134,11 +134,11 @@ trait ViewTest extends IntegrationTest {
       .filter(keys => !exclusionKeys.contains(keys))
       .toSet
 
-    val allMsgVals = msgFile.filterKeys(msgKeys)
+    val allMsgVals = msgFile.view.filterKeys(msgKeys)
       .values
       .toList
 
-    val uniqueMsgVals = msgFile.filterKeys(msgKeys)
+    val uniqueMsgVals = msgFile.view.filterKeys(msgKeys)
```

Comment:
can we replace this with `val uniqueMsgVals = allMsgVals.distinct`

Reply:
Resolved